packages/flutter/lib/src/material/button_theme.dart

```diff
@@ -0,0 +1,265 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/widgets.dart';
+
+import 'flat_button.dart';
+import 'raised_button.dart';
+import 'theme.dart';
```

Comment:
as far as i can tell you don't need to import flat_button and raised_button here